### PR TITLE
Added ipv6 to test setup

### PR DESCRIPTION
**Pull request by zealws**
Enable ipv6 packet forwarding and add ipv6 addresses and routes to the test network namespaces.

### CODE REVIEW

File: utils/test-setup.sh [hunk 1]
```diff
@@ -62,12 +63,20 @@ ip link set dev lan1 master br0
...
+ip netns exec srv ip -4 addr add dev srv0 192.168.4.2/24 broadcast 192.168.4.255
 
-# Add default routes so that out-of-network IPs will be forwarded.
-ip netns exec cli ip route add default via 192.168.3.1 dev cli0
-ip netns exec srv ip route add default via 192.168.4.1 dev srv0
+# assign IPv6 addresses to inside interfaces
+ip netns exec cli ip -6 addr add dev cli0 2620:10d:c081:1101:1::2/80
+ip netns exec atc ip -6 addr add dev lan0 2620:10d:c081:1101:1::1/80
+ip netns exec atc ip -6 addr add dev wan0 2620:10d:c081:1101:2::1/80
+ip netns exec srv ip -6 addr add dev srv0 2620:10d:c081:1101:2::2/80
```
Comment (zfjagann) [inline utils/test-setup.sh]: Probably want to switch these to somewhere in `fc00::`

**Comment by zfjagann** [comment]
Don't merge this one yet. Want to move the addresses to `fc00::`

**Comment by zfjagann** [comment]
Ready to merge now.

**Comment by chantra** [comment]
got to love network namespaces!

**Pull request merged**